frontend/packages/console-shared/src/utils/utils.ts

```diff
@@ -26,3 +26,6 @@ export const createLookup = <A extends K8sResourceKind>(
   }
   return {};
 };
+
+export const prefixedID = (idPrefix: string, id: string) =>
+  idPrefix && id ? `${idPrefix}-${id}` : null;
```

Comment:
To make the function "safe", what about returning at least one of the trueish param if any?

Reply:
Is it enough to be unique?

Reply:
No it is not (in case some mistake happens or we get a null entity). Please see the use case in `details-page.tsx`

Reply:
ok, fail fast ... Makes sense